tembo-cli/src/cmd/apply.rs

```diff
@@ -501,6 +539,28 @@ pub fn get_instance_settings(
         }
     }
 
+    if let Some(set_arg) = set_arg {
```

Comment:
Can you move this to a separate function with an appropriate name that you can call from here? Smaller functions the better.

Reply:
I would also move the overlay logic in this function to a separate function.